core/server/models/newsletter.js

```diff
@@ -72,6 +72,10 @@ const Newsletter = ghostBookshelf.Model.extend({
         return query;
     }
 }, {
+    orderDefaultRaw: function (options) {
+        return 'sort_order ASC, created_at ASC, id ASC';
```

Comment:
Strictly this shouldn't be necessary. But I'll raise a separate issue. The implementation of `findAll` should use orderDefaultOptions but it's only used in `findPage`

Reply:
Tracked here: https://github.com/TryGhost/Toolbox/issues/310

Reply:
Perfect! I tried to solve it in here (https://github.com/TryGhost/Ghost/pull/14579/commits/80f35dacc02d3cd1505a1e6702efd9b5362a8bc4), but it wasn't working so I just reverted. Better to fix in a different PR indeed